### ISSUE TITLE
Multi-line string rendering support

### ISSUE THREAD

**Issue by Tornadoally** [Redninja106/simulationframework#47]
When using Environment.NewLine or \n and such, the text renderer will just show an unknown character symbol. Would be nice for this to just wrap the text instead. You could also add an overload to the text rendering which takes in a max length and wraps for you, perhaps with different wrapping modes such as to the nearest word or even character. Thanks!

**Comment by Redninja106** [comment]
good idea. It's already possible to implement something like this using the `ICanvas.MeasureText` overloads which accept a `maxLength` argument, but it's definitely not easy. I'm considering adding a new method to the canvas called something like `DrawMultilineText` for this, since `DrawText` is already complex, especially with the recent addition of `TextAnchor`.
